src/Home/ShareLink.tsx

```diff
@@ -34,6 +35,7 @@ export const ShareLink: FunctionComponent = () => {
     return null
   }
 
+  const shareIconSize = 22
```

Comment:
Should this be included in the Iconography styles? e.g. `Iconography.tiny`

Reply:
I paired with @devinjameson on this, and he was comfortable with this being a one-off rather than redoing the scope of the iconography module, so I defer to him!